src/Mvc/Mvc.Core/src/ModelBinding/Validation/ValidationVisitor.cs

```diff
@@ -144,15 +144,6 @@ public class ValidationVisitor
         /// </summary>
         public bool ValidateComplexTypesIfChildValidationFails { get; set; }
 
-        /// <summary>
-        ///  Gets or sets a value that determines if <see cref="ValidationVisitor"/> can short circuit validation when a model
-        ///  does not have any associated validators.
-        /// </summary>
-        /// <value>The default value is <see langword="true"/>.</value>
-        /// <remarks>This property is currently ignored.</remarks>
-        [Obsolete("This property is deprecated and is no longer used by the runtime.")]
-        public bool AllowShortCircuitingValidationWhenNoValidatorsArePresent { get; set; } = true;
```

Comment:
@JeremySkinner I believe you're using ValidationVisitor in FluentValidation. Wanted to run this change by you in case you're using this API.

Reply:
@captainsafia looks good to me, I’m not using this property, so it’s removal won’t affect FluentValidation. Thanks for looping me in!